cfbs.json

```
@@ -788,6 +788,20 @@
         "append enable.cf services/init.cf"
       ]
     },
+    "promise-type-docker-compose": {
+      "description": "Promise type to manage docker compose installation",
```

It does not install, docker compose, right?

```suggestion
      "description": "Promise type to manage containers using docker compose",
```

---

Right is does not install `docker compose`